docs/_docs/configuration/options.md

```diff
@@ -352,17 +352,26 @@ before your site is served.
     <tr class="setting">
       <td>
         <p class="name"><strong>Base URL</strong></p>
-        <p class="description">Serve the website from the given base URL</p>
+        <p class="description">Serve the website from the given base URL.</p>
       </td>
       <td class="align-center">
         <p><code class="option">baseurl: URL</code></p>
         <p><code class="flag">--baseurl URL</code></p>
       </td>
     </tr>
+    <tr class="setting">
+      <td>
+        <p class="name"><strong>Live Reload</strong></p>
+        <p class="description">Reload the site automatically on the browser when content is edited.</p>
+      </td>
+      <td class="align-center">
+        <p><code class="flag">--livereload</code></p>
```

Comment:
```suggestion
        <p><code class="option">livereload: true</code></p>
        <p><code class="flag">--livereload</code></p>
```

Reply:
You can also specify that option in a config file (.e.g. `_config_local.yml`)

Reply:
Oh, I wasn't aware of that, and it's super useful for our use case. Thanks!